glide.yaml

```diff
@@ -23,14 +23,17 @@ import:
   - pkg/api/v1/helper
 - package: k8s.io/apimachinery
   version: kubernetes-1.8.2
+  subpackages:
+  - pkg/runtime
 - package: k8s.io/api
   version: kubernetes-1.8.2
 - package: k8s.io/client-go
   version: kubernetes-1.8.2
 - package: github.com/portworx/sched-ops
-  version: master
-  vcs: git
+  version: master 
   repo: git@github.com:portworx/sched-ops.git
+  subpackages:
```

Comment:
why do we not need to subpackage task?

Reply:
This was autogenerated using "glide up".

Reply:
I thought glide up does not update the glide.yaml file. It populates it only for the first time.
Can you try removing subpackages (or add task) and running glide up again?

Reply:
glide.lock had both k8s and tasks package so it was fine. I've removed the subpackages from here